agent/04_agent_configure.sh

```diff
@@ -83,8 +83,14 @@ metadata:
   name: test-agent-cluster-install
   namespace: cluster0
 spec:
+EOF
+if [[ "$NUM_MASTERS" > "1" ]]; then
```

Comment:
Maybe it would be slightly more clear if the condition was set againt `AGENT_CONFIG` (ie when it's not compact)

Reply:
I would suggest to put the condition around the `set_api_and_ingress_vip` function and set the empty value for api_vip and egress_vip when the case is SNO.
For Example:
```
if [[ $AGENT_CONFIG  =~ "SNO" ]]; then
   echo "Setting empty value for api_vip and ingress_vip as these are not required for SNO cluster"
   API_VIP=""
   INGRESS_VIP=""
else
   set_api_and_ingress_vip 
fi
```